pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -451,18 +453,31 @@ func ExecuteDDCmd(ledName string) {
 }
 
 func doDD(count int) {
```

Comment:
Can we change the name of the func to make it mnemonic?

Reply:
Yeah like uncachedDiskRead() or lightDiskLED()